src/EventListenerService/Modules/VoiceModule.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.EntityFrameworkCore;
+using NetCord.Services.ApplicationCommands;
+
+[SlashCommand("voice", "Voice command")]
+public class VoiceModule : ApplicationCommandModule<ApplicationCommandContext>
+{
+  [SubSlashCommand("status", "Update channel status when you join a channel")]
+  public class VoiceStatusModule(IServiceScopeFactory scopeFactory) : ApplicationCommandModule<ApplicationCommandContext>
+  {
+    [SubSlashCommand("set", "Status to set")]
+    public async Task<string> SetStatusAsync(
+      [SlashCommandParameter(Description = "Emoji")] string emoji)
```

Comment:
there's no validation if emoji contains a valid emoji
as of now it can be any string